### PR TITLE
Issue13

### PR DESCRIPTION
**Pull request by jeremiahliao**
form layout completed
Will do these when linking frontend and backend:
 - photo display and upload to database needs to be fixed
 - postContact function will be changed to make a request successfully

form input is stored into contact object
{
      photo: "",
      college: "",
      major: "",
      firstName: "",
      lastName: "",
      email: "",
      phoneNumber: "",
      birthday: "",
      address: {
          country: "",
          street: "",
          city: "",
          region: "",
          postalCode: ""
      },
      social:{
          facebook: "",
          instagram: "",
          snapchat: "",
          twitter: "",
          linkedin: "",
          discord: "",
          github: "",
          spotify: ""
      }
}

### CODE REVIEW

**Review by shua1090 — CHANGES_REQUESTED** [review]
Hey Jeremiah, this looks great! When you get into the actual REST API stuff, can you flatten your request from this (what you sent earlier):
{
photo: "",
college: "",
major: "",
firstName: "",
lastName: "",
email: "",
phoneNumber: "",
birthday: "",
address: {
country: "",
street: "",
city: "",
region: "",
postalCode: ""
},
social:{
facebook: "",
instagram: "",
snapchat: "",
twitter: "",
linkedin: "",
discord: "",
github: "",
spotify: ""
}
}

to 

{
photo: "",
college: "",
major: "",
firstName: "",
lastName: "",
email: "",
phoneNumber: "",
birthday: "",
country: "",
street: "",
city: "",
region: "",
postalCode: "",
facebook: "",
instagram: "",
snapchat: "",
twitter: "",
linkedin: "",
discord: "",
github: "",
spotify: ""
}

Basically no separate address/socials section, all in one thing. Other than that, looks great!

**Comment by jeremiahliao** [comment]
changed request body

**Review by shua1090 — APPROVED** [review]
image

looks great!

**Review by reakunen — COMMENTED** [review]
delicious.

**Review by reakunen — APPROVED** [review]
